opal/mca/pmix/pmix2x/Makefile.am

```diff
@@ -10,6 +10,8 @@
 # $HEADER$
 #
 
+if !OPAL_PMIX_USE_EXTERNAL
```

Comment:
But if you put it in the conditional, then if you `make dist`, that file may not be included in the tarball.  That's why the DIST-level stuff *always* has to be filled properly, regardless of which components decide to build/not build.  Make sense?